utils/search/backends/solr555pysolr.py

```diff
@@ -259,13 +259,13 @@ def convert_sound_to_search_engine_document(self, sound):
             document["type"] = "wav"
         else:
             document["type"] = sound.type
-        document["name"] = remove_control_chars(getattr(sound, "original_filename"))
-        document["description"] = remove_control_chars(getattr(sound, "description"))
-        document["tag"] = list({t.lower() for t in getattr(sound, "tag_array")})
+        document["name"] = remove_control_chars(sound.original_filename)
+        document["description"] = remove_control_chars(sound.description)
+        document["tag"] = list({t.lower() for t in sound.tag_array})
```

Comment:
I'm unsure about these changes, as I mentioned in a previous comment django does magic with getattr vs model.x when it's a OneToOne, however these fields all look like actual fields in our sound object. I guess the original idea here was to return None if it didn't exist? We should try and do a full reindex to check that it works as expected

Reply:
I did a reindex and it seems to work, but only with the 5000 local files. I think in the past we supported creating these documents from different types of objects, but since we added `bulk_query_solr` we know the type of object that we get and we could be sure that it has such fields. I would keep the changes as the code becomes more readable.